### PR TITLE
Move common MP Config message to a common bundle

### LINKED ISSUE #25590: Create separate NLS message bundle for SmallRye Config 3.x
When the SmallRye Config 3 bundle was created, we just copied the existing NLS bundle without changing the message codes. This needs to be made unique.

`dev/io.openliberty.io.smallrye.config3/resources/io/openliberty/io/smallrye/config/resources/SmallryeConfigMessages.nlsprops`

https://github.com/OpenLiberty/open-liberty/pull/25587/files#diff-afdd57e4ce4de7757e84e29b5b543e745ab2ef23fc552e61610ec5780b014b42

### PR DESCRIPTION
**Pull request by tevans78**
fixes https://github.com/OpenLiberty/open-liberty/issues/25590

There was a duplicated message used by our integration of both SmallRye Config 2.x and 3.x. This PR moves that message to a common bundle so that it only exists in one place.

### CODE REVIEW

**Comment by tevans78** [comment]
#build

File: dev/io.openliberty.microprofile.config.internal.common/resources/io/openliberty/microprofile/config/internal/common/resources/MPConfigCommonMessages.nlsprops [hunk 1]
```diff
@@ -1,28 +1,29 @@
...
-WARNING_UPDATED_CONFIG_PROPERTY_NOT_USED_CWWKC0651.explanation=The configuration value was accessed during a checkpoint action and updated before the server process got restored. Depending on how the application accesses the configuration, the updated value might not be used.
-WARNING_UPDATED_CONFIG_PROPERTY_NOT_USED_CWWKC0651.useraction=Avoid accessing configuration properties during checkpoint.
+WARNING_UPDATED_CONFIG_PROPERTY_NOT_USED_CWMCG0651=CWWKC0651W: The MicroProfile configuration value for the {0} key has changed since the checkpoint action completed on the server. If the value of the key changes after the checkpoint action, the application might not use the updated value.
+WARNING_UPDATED_CONFIG_PROPERTY_NOT_USED_CWMCG0651.explanation=The configuration value was accessed during a checkpoint action and updated before the server process got restored. Depending on how the application accesses the configuration, the updated value might not be used.
+WARNING_UPDATED_CONFIG_PROPERTY_NOT_USED_CWMCG0651.useraction=Avoid accessing configuration properties during checkpoint.
```
Comment (Azquelt) [inline dev/io.openliberty.microprofile.config.internal.common/resources/io/openliberty/microprofile/config/internal/common/resources/MPConfigCommonMessages.nlsprops]: I don't think you want to change the message code here.
- This is a Config + InstantOn message
- There's no corresponding update to the code or to the other language NLS files

**Comment by Azquelt** [comment]
The only tests which deliberately provoke this message are in `io.openliberty.checkpoint_fat_mpconfig`, do we need an extra build to run that?

File: dev/io.openliberty.microprofile.config.internal.common/resources/io/openliberty/microprofile/config/internal/common/resources/MPConfigCommonMessages.nlsprops (hunk 1 above)
Comment (tevans78) [reply dev/io.openliberty.microprofile.config.internal.common/resources/io/openliberty/microprofile/config/internal/common/resources/MPConfigCommonMessages.nlsprops]: ack! no, that is a mistake!

**Comment by tevans78** [comment]
> The only tests which deliberately provoke this message are in `io.openliberty.checkpoint_fat_mpconfig`, do we need an extra build to run that?

Yes, I think so

**Comment by tevans78** [comment]
#build

**Comment by tevans78** [comment]
> > The only tests which deliberately provoke this message are in `io.openliberty.checkpoint_fat_mpconfig`, do we need an extra build to run that?
> 
> Yes, I think so

An InstantOn PB - https://wasrtc.hursley.ibm.com:9443/jazz/web/projects/WS-CD#action=com.ibm.team.build.viewResult&id=_RlfIQHNQEe6LyICjST0JEw

**Review by Azquelt — DISMISSED** [review]

**Comment by posmith** [comment]
L2 message review completed.

**Comment by tevans78** [comment]
I think I found and fixed the problem which caused the InstantOn test failure. Started a new InstantOn test build:
https://wasrtc.hursley.ibm.com:9443/jazz/web/projects/WS-CD#action=com.ibm.team.build.viewResult&id=_yW3NIHdBEe6p7N33NVgeoQ

**Comment by tevans78** [comment]
#build

**Review by Azquelt — APPROVED** [review]

**Comment by tevans78** [comment]
InstantOn build was green too.

**Comment by tevans78** [comment]
#libby

**Pull request merged**